autoload/lsc/reference.vim

```diff
@@ -143,6 +143,15 @@ function! s:showHover(result) abort
       let l:lines += split(item, "\n")
     endif
   endfor
+  if exists('g:lsc_preview_hover_popup') && exists('*popup_atcursor')
```

Comment:
I think this would enable pop up window even if user set option to 0
```vimscript
let g:lsc_preview_hover_popup = 0
```

Reply:
woops. 
thank you checking!
I use `get` instead `exists`.